### DISCUSSION TITLE
Is there a way to hide databases at the left?

### DISCUSSION THREAD

**Discussion by davinhun** [YetiForceCompany/YetiForceCRM#16869] [How to] [answered]
I would like to hide databases tab for the users. I haven't found any privileges in the profiles to hide it.

![image](https://user-images.githubusercontent.com/20862325/217384296-282da860-9394-4678-9e85-327b6e7143b2.png)

![image](https://user-images.githubusercontent.com/20862325/217384476-c541d682-4bdb-441f-bfd1-f9a0c85eac95.png)

How can I do that?

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
this is managed in the menu panel
https://gitdeveloper.yetiforce.com/index.php?module=Menu&view=Index&parent=Settings
